### DISCUSSION TITLE
More query metadata

### DISCUSSION THREAD

**Discussion by hwillson** [apollographql/apollo-client-devtools#369] [Ideas]
Migrated from: https://github.com/apollographql/apollo-client-devtools/issues/4

In addition to seeing the Query String and Variables, it would be great if you could also see things like the query Options. For queries with `pollIntervals`, it would also be very cool if there was a pause button.

**Comment by jerelmiller** [comment]
With the release of [4.15.0](https://github.com/apollographql/apollo-client-devtools/releases/tag/v4.15.0) an "Options" tab has been added when viewing a query to see that query's options:

![image](https://github.com/apollographql/apollo-client-devtools/assets/565661/5759ad21-7f22-4afa-a998-17593b7db9e8)


I'll leave this open for now to ensure the polling capability is captured, but at least we can see the options specified now 🙂
